providers/openfeature-provider-flagd/README.md

```diff
@@ -64,8 +65,6 @@ The default options can be defined in the FlagdProvider constructor.
 <!-- not implemented
 | target_uri               | FLAGD_TARGET_URI               | alternative to host/port, supporting custom name resolution | string    | null                | rpc & in-process |
 | socket_path              | FLAGD_SOCKET_PATH              | alternative to host port, unix socket                       | String    | null                | rpc & in-process |
-| cert_path                | FLAGD_SERVER_CERT_PATH         | tls cert path                                               | String    | null                | rpc & in-process |
-| max_event_stream_retries | FLAGD_MAX_EVENT_STREAM_RETRIES | int                                                         | 5         | rpc                 |
```

Comment:
any reason you removed this line?

Reply:
those are just comments, which should make it easier in the future, for people to add the values to the table. As i added cert_path tot the table, i removed it from the comments.

Max_event_retries is an option which is still supported by the java sdk, but deprecated and not part of the flagd provider spec, hence that i removed it too

Reply:
Gotcha, was only curious about max event retries